middleware_jwt_test.go

```diff
@@ -179,10 +179,10 @@ var jwtWithCentralDefNoPolicyBaseField = `
 
 `
 
-const JWTSECRET = "9879879878787878"
+const jwtSecret = "9879879878787878"
 
 // openssl genrsa -out app.rsa
-const JWTRSA_PRIVKEY = `
+const jwtRSAPrivKey = `
```

Comment:
@lonelycode note that what you're referring to is grouping similar things by name. This applies to vars, consts, funcs, whatever it is. It doesn't tell you anything about the names being consts.